src/App.js

```diff
@@ -65,33 +70,36 @@ function App() {
       jsPsych.data.addProperties({
         participant_id: participantId,
         study_id: studyId,
-        timestamp: timestamp,
         start_date: startDate,
       });
       setLogin(loggedIn)
     },
-    [startDate, timestamp],
+    [startDate],
   )
 
   // Login logic
   useEffect(() => {
     // For testing and debugging purposes
-    console.log("MTURK? :" + MTURK);
-    console.log("ELECTRON? :" + IS_ELECTRON);
-    console.log("AT_HOME? :" + AT_HOME);
-    console.log("FIREBASE? :" + FIREBASE);
-
+    console.log("Outside Turk:", jsPsych.turk.turkInfo().outsideTurk);
+    console.log("Turk:", MTURK);
+    console.log("Firebase:", FIREBASE);
+    console.log("Prolific:", PROLIFIC);
+    console.log("Electron:", IS_ELECTRON);
+    console.log("Video:", VIDEO);
+    console.log("Volume:", VOLUME);
+    console.log("Event Marker:", USE_EVENT_MARKER);
+    console.log("Photodiode:", USE_PHOTODIODE);
     // If on desktop
     if (IS_ELECTRON) {
       const electron = window.require("electron");
       const renderer = electron.ipcRenderer;
       setRenderer(renderer);
       // If at home, fill in fields based on environment variables
       const credentials = renderer.sendSync("syncCredentials");
-      if (credentials.envParticipantId !== null) {
+      if (credentials.envParticipantId !== null && credentials.envParticipantId !== undefined) {
```

Comment:
can we instead check for the positive case (e.g. `if (credentials.envParticipantId) ...` or does that miss one of these edge cases?